admin_guide/high_availability.adoc

```diff
@@ -252,23 +282,95 @@ ip addr show $interface | grep -i MULTICAST
 echo "Check multicast groups ... "
 ip maddr show $interface | grep 224.0.0 | grep $interface
 
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
 EOF
 done;
 ----
 ====
 
-. Depending on your environment policies, you can either reuse the *router*
-service account created previously or create a new *ipfailover* service account.
+[[options-environment-variables]]
+=== Command Line Options and Environment Variables
+
+.Command Line Options and Environment Variables
+[cols="1a,3a,1a,4a",options="header"]
+|===
+
+| Option | Variable Name | Default | Notes
+
+|--watch-port
+|`*OPENSHIFT_HA_MONITOR_PORT*`
+|80
+|The ipfailover pod will test that it can open a TCP connection to this port on each VIP. If it can, then the
+service is considered to be running. When set to 0 the test always passes.
+
+|--interface
+|`*OPENSHIFT_HA_NETWORK_INTERFACE*`
+|
+|The interface name for the ip failover to use to send VRRP traffic. By default, eth0 is used.
+
+|
+|`*OPENSHIFT_HA_REPLICA_COUNT*`
+|2
+|This must match spec.replicas in the Deployment Config
+
+|--virtual-ips
+|`*OPENSHIFT_HA_VIRTUAL_IPS*`
+|
+|The list of IP address ranges to replicate. This must be provided. E.g., 1.2.3.4-6,1.2.3.9
```

Comment:
The aggregate of all of the VIPs in all of tie ipfailover configs must be less than 255